src/scss/blocks/_button.scss

```diff
@@ -57,6 +58,10 @@
 	&.is-style-outline {
 		> .wp-block-button__link,
 		form .wp-block-button__link {
+			background: var(--wp--preset--color--base);
+			border: 1px solid currentcolor;
```

Comment:
Question: Do we need to make changes to the border color on hover as well? Solid buttons seem to become a darker shade of the same color on hover, while outline buttons appear to become black.

Reply:
Thanks for clarifying!